api/sixtyfps-cpp/docs/index.rst

```diff
@@ -39,6 +39,8 @@ Welcome to SixtyFPS C++'s documentation!
 
    markdown/debugging_techniques.md
 
+   cpp_migration.md
```

Comment:
not sure if we need to call that cpp_migration.md or just migration.md
We are anyway in the sixtyfps-cpp folder after all.
(on the other hand, we would end up with several migration.md in the project. Which i don't know if it is a problem (we have many lib.rs files already))

Reply:
That was exactly what I was thinking as well and motivating me to this prefix :)